general/calendars/outlook/outlook_calendar.rst

```diff
@@ -0,0 +1,57 @@
+=====================================
+Synchronization with Outlook Calendar
+=====================================
+
+Life is a lot easier when you are able to manage all your calendars in one single place. Therefore,
+now Odoo also allows you to synchronize your Outlook Calendar with the Calendar application.
+
+To start, `Register your application with the Microsoft identity platform
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_ making
+sure to **follow all the steps described in the guidelines**.
+
+.. important::
+   For the **Redirect URI restrictions**, copy your Odoo server URI followed by
+   ``/microsoft_account/authentication``. Example:
+
+   .. image:: media/redirect_URIs.png
+      :align: center
+      :alt: Odoo's server URI that is accepted when microsoft returns authentication
+
+Once you are finished, copy:
+
+- Client ID:
+
+.. image:: media/application_client_ID.png
+   :align: center
+   :alt: Client ID token to be copied from Microsoft
+
+- Client Secrets:
+
+.. image:: media/client_secrets.png
+   :align: center
+   :alt: Client Secret token to be copied from Microsoft
+
+Now, activate the *Outlook Calendar* feature in *Settings* and paste those key pieces of information.
```

Comment:
specify that it is in the "general settings", as there are no "calendar settings".
Also, I read "key pieces of information" as "really sensitive information". I think you simply meant "keys"?